en, zh: refine docs: back up and restore data using BR (#1559)
pingcap/docs-tidb-operator: en/backup-to-gcs-using-br.md

        # objectAcl: private
    ```

>     When configuring the `backup-aws-s3.yaml`, note the following:

Author: Good catch 👍

    - Since TiDB Operator v1.1.6, if you want to back up data incrementally, you only need to specify the last backup timestamp `--lastbackupts` in `spec.br.options`. For the limitations of incremental backup, refer to [Use BR to Back up and Restore Data](https://docs.pingcap.com/tidb/stable/backup-and-restore-tool#back-up-incremental-data).
    - Some parameters in `spec.br` are optional, such as `logLevel` and `statusAddr`. For more information about BR configuration, refer to [BR fields](backup-restore-overview.md#br-fields).

>     - Some parameters in `spec.gcs` can be ignored, such as `location`, `objectAcl`, and `storageClass`. For more information about GCS configuration, refer to [GCS fields](backup-restore-overview.md#gcs-fields).

Author: ```suggestion
    - Some parameters in `spec.gcs` are optional, such as `location`, `objectAcl`, and `storageClass`. For more information about GCS configuration, refer to [GCS fields](backup-restore-overview.md#gcs-fields).
```
